src/SemiclassicalOrthogonalPolynomials.jl

```diff
@@ -260,56 +268,110 @@ function semijacobi_ldiv(P::SemiclassicalJacobi, Q)
     (P \ R) * _p0(R̃) * (R̃ \ Q)
 end
 
-# returns conversion operator from SemiclassicalJacobi P to SemiclassicalJacobi Q in a single step via decomposition.
-semijacobi_ldiv_direct(Q::SemiclassicalJacobi, P::SemiclassicalJacobi) = semijacobi_ldiv_direct(Q.t, Q.a, Q.b, Q.c, P)
-function semijacobi_ldiv_direct(Qt, Qa, Qb, Qc, P::SemiclassicalJacobi)
-    (Qt ≈ P.t) && (Qa ≈ P.a) && (Qb ≈ P.b) && (Qc ≈ P.c) && return SymTridiagonal(Ones(∞),Zeros(∞))
-    Δa = Qa-P.a
-    Δb = Qb-P.b
-    Δc = Qc-P.c
-    M = Diagonal(Ones(∞))
-    if iseven(Δa) && iseven(Δb) && iseven(Δc)
-        M = qr(P.X^(Δa÷2)*(I-P.X)^(Δb÷2)*(Qt*I-P.X)^(Δc÷2)).R
+"""
+    semijacobi_ldiv_direct(Q::SemiclassicalJacobi, P::SemiclassicalJacobi)
+
+Returns conversion operator from SemiclassicalJacobi `P` to SemiclassicalJacobi `Q` in a single step via decomposition. 
+Numerically unstable if the parameter modification is large. Typically one should instead use `P \\ Q` which is equivalent to `semijacobi_ldiv(P,Q)` and proceeds step by step.
+"""
+function semijacobi_ldiv_direct(Q::SemiclassicalJacobi, P::SemiclassicalJacobi)
+    (Q.t ≈ P.t) && (Q.a ≈ P.a) && (Q.b ≈ P.b) && (Q.c ≈ P.c) && return SymTridiagonal(Ones(∞),Zeros(∞))
+    Δa = Q.a-P.a
+    Δb = Q.b-P.b
+    Δc = Q.c-P.c
+    # special case (Δa,Δb,Δc) = (2,0,0)
+    if isone(Δa/2) && iszero(Δb) && iszero(Δc)
+        M = qr(P.X^(Δa÷2)).R
+        return ApplyArray(*, Diagonal(sign.(view(M,band(0))).*Fill(abs.(1/M[1]),∞)), M) # match normalization choice P_0(x) = 1
+    # special case (Δa,Δb,Δc) = (0,2,0)
+    elseif iszero(Δa) && isone(Δb/2) && iszero(Δc)
+        M = qr((I-P.X)^(Δb÷2)).R
+        return ApplyArray(*, Diagonal(sign.(view(M,band(0))).*Fill(abs.(1/M[1]),∞)), M)
+    # special case (Δa,Δb,Δc) = (0,0,2)
+    elseif iszero(Δa) && iszero(Δb) && isone(Δc/2)
+        M = qr((Q.t*I-P.X)^(Δc÷2)).R
         return ApplyArray(*, Diagonal(sign.(view(M,band(0))).*Fill(abs.(1/M[1]),∞)), M)
-    elseif isone(Δa) && iszero(Δb) && iszero(Δc) # special case (Δa,Δb,Δc) = (1,0,0)
+    # special case (Δa,Δb,Δc) = (-2,0,0)
+    elseif isone(-Δa/2) && iszero(Δb) && iszero(Δc)
```

Comment:
Seriously, why do you keep taking matrices to the power 1??

Reply:
Leftovers from copy and pasting it out of general cases probably. 😅 I'll sort it.